tests/integration/targets/dcnm_vrf/tests/dcnm/merged.yaml

```diff
@@ -310,15 +317,14 @@
       - ip_address: "{{ ansible_switch1 }}"
       - ip_address: "{{ ansible_switch2 }}"
         vrf_lite:
-          - interface: "{{ ansible_int1 }}"
       deploy: true
   register: result
   ignore_errors: yes
 
 - assert:
     that:
     - 'result.changed == false'
-    - '"peer_vrf is mandatory under attach VRF LITE parameters" in result.msg'
+    - '"Please specifiy atleast one VRF LITE parameter in attach" in result.msg'
```

Comment:
`/atleast/at least` After you change the code :)

Reply:
Changes done as suggested